{
  "comments": [
    {
      "key": {
        "uuid": "9a561d44_824b8486",
        "filename": "yardstick/common/exceptions.py",
        "patchSetId": 3
      },
      "lineNbr": 435,
      "author": {
        "id": 7363
      },
      "writtenOn": "2018-09-04T10:21:14Z",
      "side": 1,
      "message": "one too many, will fix",
      "revId": "ede6eeb4f34ce607cd13eefb29612f77a2f4813a",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "9a561d44_42820c95",
        "filename": "yardstick/tests/unit/network_services/vnf_generic/vnf/test_tg_landslide.py",
        "patchSetId": 3
      },
      "lineNbr": 324,
      "author": {
        "id": 7363
      },
      "writtenOn": "2018-09-04T10:21:14Z",
      "side": 1,
      "message": "miss during patch rebase. will fix",
      "revId": "ede6eeb4f34ce607cd13eefb29612f77a2f4813a",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}